.github/workflows/release.yml

```diff
@@ -5,10 +5,21 @@ on:
     branches:
       - "release/**"
 
+  # FIXME: Temporarily build pull requests.
+  pull_request:
```

Comment:
Don't forget to resolve this before merging

Reply:
Thank you, I would've surely forgot. :)